deepforest/utilities.py

```diff
@@ -65,44 +75,37 @@ def use_bird_release(
     output_path = os.path.join(save_dir, prebuilt_model + ".pt")
 
```

Comment:
Here the bird.pt file is not been created, so all the test cases file failed.

Reply:
Try to find out why it is not created